users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -1122,6 +1122,200 @@ def run_mel_audio_perturbation_from_checkpoint():
+            ),
+            "bs10k_scf_stft_mask_5_8_smaller_frames": dict(
+                returnn_args={
+                    **returnn_args,
+                    "specaug_stft": {"max_feature": 8, "frame_size": 200, "frame_shift": 80 },
+                    "batch_size": 10000,
+                    "extra_args": {
+                        "conv_pad_seq_len_to_power": 1.5,
+                    },
+                },
+                feature_args=feature_args,
+                lr_args=lr_args,
+                report_args={"batch_size": "10k"},
+            ),
+            "bs10k_scf_stft_mask_5_8_smaller_frames_and_fft": dict(
+                returnn_args={
+                    **returnn_args,
+                    "specaug_stft": {"max_feature": 8,  "frame_size": 200, "frame_shift": 80, "fft_size": 256},
+                    "batch_size": 10000,
+                    "extra_args": {
+                        "conv_pad_seq_len_to_power": 1.5,
+                    },
+                },
+                feature_args=feature_args,
+                lr_args=lr_args,
+                report_args={"batch_size": "10k"},
+            ),
+            "bs10k_mel_stft_mask_5_8_smaller_frames_and_fft": dict(
+                returnn_args={
+                    **returnn_args,
+                    "specaug_stft": {"max_feature": 8, "frame_size": 200, "frame_shift": 80, "fft_size": 256},
+                    "batch_size": 10000,
+                    "extra_args": {
+                        "conv_pad_seq_len_to_power": 1.5,
+                    },
+                },
+                feature_args=feature_args_lgm,
+                lr_args=lr_args,
+                report_args={"batch_size": "10k"},
+            ),
+            "bs10k_scf_stft_mask_5_4_smaller_frames_and_fft": dict(
+                returnn_args={
+                    **returnn_args,
+                    "specaug_stft": {"max_feature": 4, "frame_size": 200, "frame_shift": 80, "fft_size": 256},
+                    "batch_size": 10000,
+                    "extra_args": {
+                        "conv_pad_seq_len_to_power": 1.5,
+                    },
+                },
+                feature_args=feature_args,
+                lr_args=lr_args,
+                report_args={"batch_size": "10k"},
+            ),
+            "bs10k_mel_stft_mask_5_4_smaller_frames_and_fft": dict(
+                returnn_args={
+                    **returnn_args,
+                    "specaug_stft": {"max_feature": 4, "frame_size": 200, "frame_shift": 80, "fft_size": 256},
+                    "batch_size": 10000,
+                    "extra_args": {
+                        "conv_pad_seq_len_to_power": 1.5,
+                    },
+                },
+                feature_args=feature_args_lgm,
+                lr_args=lr_args,
+                report_args={"batch_size": "10k"},
+            ),
+        },
+        num_epochs=450,
+        evaluation_epochs=[24, 350, 390, 400, 410, 450],
```

Comment:
Did you already run these experiments with the proper hash? Otherwise we could add more epochs, also for recognition below.
```suggestion
        evaluation_epochs=[24, 350, 390, 400, 410, 420, 430, 440, 450],
```

Reply:
I did not run them with the correct hashes. I can add the checkpoints, but the training folder will become very large.